glpi/Dockerfile

```diff
@@ -70,6 +70,11 @@ RUN apt-get update \
   && docker-php-ext-enable apcu \
   && echo "apc.enable=1" >> /usr/local/etc/php/conf.d/docker-php-ext-apcu.ini \
   \
+  # Install Redis PHP extension.
+  && pecl install redis \
+  && docker-php-ext-enable redis \
+  && echo "redis.enable=1" >> /usr/local/etc/php/conf.d/docker-php-ext-redis.ini \
```

Comment:
not sure about this line.
In a stackoverflow topic, I see several way to enable the extension:
https://stackoverflow.com/a/31369892